### ISSUE TITLE
question about hybrid training strategy

### ISSUE THREAD

**Issue by s7ev3n** [dexmal/dexbotic#80]
Thanks for open-sourcing DM0 ! 

I am curious about where do we implement the hybrid training strategy in the paper "DM0 employs a hybrid training strategy: for embodied data, gradients from the action expert are not backpropagated to the VLM to preserve generalized representations" ?  Do you freeze VLM to achieve gradient flow ? 

Thanks

**Comment by dexmal2025** [comment]
Hi, thanks for your interest on our DM0! As for your question, the hybrid training strategy is similar to the KI (https://arxiv.org/abs/2505.23705), making the gradients from the action expert are not backpropagated to the VLM. The VLM is not freezed in all stages.

**Comment by s7ev3n** [comment]
Hi, 

Thanks very much for your reply. Does this stop gradients implemented in current codebase ? If you point me out this piece of code, that will be great, I did not locate them :)

**Comment by dexmal2025** [comment]
The hybrid training strategy is not open-sourced currently. This strategy is used during the pretraining stage of DM0. Current open-sourced version is mainly used for full finetuning.
